piwik/templates/piwik/tracking_code.html

```diff
@@ -1,12 +1,17 @@
-<!-- Piwik --> 
+<!-- Start Piwik Tracking Code -->
 <script type="text/javascript">
```

Comment:
Seems like the in HTML5 it give a Warning "The type attribute is unnecessary for JavaScript resources.", what do you think about removing it ?

Reply:
Hey, I didn't maintained this here for a very long time. Also https://github.com/jcassee/django-analytical makes a great job, so I abandoned my package.